crates/communication/src/client/parameter_subscription_manager.rs

```diff
@@ -1,5 +1,6 @@
 use std::collections::{hash_map::Entry, BTreeSet, HashMap};
 
+use color_eyre::eyre::Result;
```

Comment:
The error here is completely contained in this module. It's not part of the public interface.
I'm not sure adding a new error type with `thiserror` makes sense here.

Reply:
Think about using `Result<(), String>` we do not need `eyre` here

Reply:
We do actually because we `Try` two Results with different `SendError<T>`s in the `update_parameter_value` function.
If we wanted to use `Result<(), String>`, we'd have to `map_err` them inside the function. I'm not sure that's better than just using `eyre`.